src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CourseActivityPayRate.kt

```diff
@@ -0,0 +1,78 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import org.hibernate.Hibernate
+import java.io.Serializable
+import java.math.BigDecimal
+import java.time.LocalDate
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.FetchType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+import javax.validation.constraints.Size
+
+@Entity
+@Table(name = "COURSE_ACTIVITY_PAY_RATES")
+data class CourseActivityPayRate(
+
+  @Id
+  @ManyToOne(optional = false, fetch = FetchType.LAZY)
+  @JoinColumn(name = "CRS_ACTY_ID", nullable = false)
+  var courseActivity: CourseActivity,
+
+  @Id
+  @Column(name = "IEP_LEVEL")
+  @Size(max = 12)
+  val iepLevelCode: String,
```

Comment:
do you get different pay rates per IEP level?

Reply:
Dont know, but iep is part of the composite primary key so seems to be important